packages/sdk-v2/src/sdk/claim/sdk.ts

```diff
@@ -59,6 +60,11 @@ export class ClaimSDK extends BaseSDK {
       const url = new URL(identityUrl);
       const { searchParams } = url;
 
+      // the link will be requested to send a user to the fv-flow
+      // we want to prevent a user to have to sign again when it redirects
+      // so we store the fv-sig locally
+      await AsyncStorage.setItem("fv-sig", fvSig);
```

Comment:
this should happen in getFvSig.
getFvSig should read from storage if exists.
When will we be clearing the fv sig from localstorage? ie users connects with another wallet so we cant use sig from localstorage.
or maybe this should not happen here at all but rather the logic of storing/removing should happen by the component/logic using this sdk

Reply:
I thought it would just be removed at the end of succesfull FV. no need to keep storing it.
(not reflected this pr as that is done in a hook/screen not part of this pr)
But considering your comment:
1. could remove it on page load when there is no isVerified param (meaning came back from FV when it does exist)
2. could remove it whenever someone connects a new wallet
3. and then remove it on fvSuccess (after its no longer needed)

There is no flow where we return the FvSig (except when running getFvSig)
but considering there might be alternative flows (eg: the good-id details screen / fvDelete flow) where we just need the fv-signature not related to the verify flow I thought it would not make sense to add it to getFvSig method.

The only reason we want to store it (currently) is when someone starts the fv-flow.
And the only reason to call getLink is to start the fv-flow.
That was my consideration for putting it there.

Reply:
it cant be removed at the end of successfull FV because the caller wants to use it after FV, ie to call the get certificate api
because this is a free specific flow that we want to implement, i do think it should be part of the sdk
it seems most reasonable to me that useFVModal will return to the caller the FV sig
then the caller will have to handle any storing/deleting of the FV sig.

Reply:
so modify verify not to be void and return the fvsig instead of modifying verifying to store it in localstorage.
so after getting the certificate is in the caller context, that's why the caller should handle storing/deleting the fvsig

Reply:
I am not sure I follow?

We do verify flow > you get redirected to good id server
once you come back through a redirect from good-id, all we do is look for the isVerified param (was it succesfull or not)

I don't understand waht making 'verify not a void' would accomplish except having to move the redirect logic out of the useFvModalAction

Reply:
Applied your requested changes here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/130/commits/3a197ac5090b24af7a7c7585a377b338b4567433